src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -994,6 +1066,46 @@ class MediaBaseViewer extends BaseViewer {
         return true;
     }
 
+    handleKeydownReact(key) {
+        switch (key.toLowerCase()) {
+            case 'space':
+            case 'k':
+                this.togglePlay();
+                break;
+            case 'arrowleft':
+                this.quickSeek(-5);
+                break;
+            case 'j':
+                this.quickSeek(-10);
+                break;
+            case 'arrowright':
+                this.quickSeek(5);
+                break;
+            case 'l':
+                this.quickSeek(10);
+                break;
+            case '0':
+            case 'home':
+                this.setMediaTime(0);
+                this.renderUI();
+                break;
+            case 'arrowup':
+                this.increaseVolume();
+                break;
+            case 'arrowdown':
+                this.decreaseVolume();
+                break;
+            case 'm':
+            case 'shift+m':
+                this.toggleMute();
+                break;
+            default:
```

Comment:
Do we need to handle keys like `shift+<` in original handleKeyDown?

Reply:
We'll need to handle them once we add support for the video controls. It didn't seem worthwhile to add it just yet.